tests/testversion.pas

```diff
@@ -0,0 +1,54 @@
+program testversion;
+
+{
+
+  Test version macros/functions of SDL2 system.
+
+  This file is part of
+
+    SDL2-for-Pascal
+    Copyright (C) 2020-2022 PGD Community
+    Visit: https://github.com/PascalGameDevelopment/SDL2-for-Pascal
+
+}
+
+{$I testsettings.inc}
+
+uses
+  Classes, SysUtils, SDL2;
+
+type
+  ESDL2Error = class(Exception);
+
+var
+  CompiledVersion: TSDL_Version = (major: 0; minor: 0; patch: 0);
+  LinkedVersion: TSDL_Version = (major: 0; minor: 0; patch: 0);
+begin
+  write('Start SDL2 version test... ');
+
+  try
+    if (SDL_VERSIONNUM(1,2,3) <> 1203) then
+      raise ESDL2Error.Create('SDL_VERSIONNUM failed: 1203 expected, found: ' + IntToStr(SDL_VERSIONNUM(1,2,3)));
```

Comment:
1. Consider assigning `SDL_VERSIONNUM()` to a variable, instead of calculating it twice.
2. Consider using [`WriteStr()`](https://www.freepascal.org/docs-html/current/rtl/system/writestr.html) here.

Reply:
I wonder, what would be the benefit of WriteStr here? 

Best regards